src/link/abstractlink.h

```diff
@@ -250,6 +250,6 @@ class AbstractLink : public QObject
 
 private:
     bool _autoConnect;
-    LinkType _type;
     QString _name;
```

Comment:
should _name be initialized in the constructor?

Reply:
Yep, check: https://github.com/bluerobotics/ping-viewer/issues/808